packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -72,6 +74,8 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
             initializationMethodId = methodId;
         } else if (this._isPaylater) {
             initializationMethodId = 'paylater';
+        } else if (this._isVenmo) {
```

Comment:
```suggestion
        } else if (this._isVenmo && isVenmoEnabled) {
```

Reply:
This condition was checked https://github.com/bigcommerce/checkout-sdk-js/blob/9ea80891c19393fd375fd1f65c328b2c432d8103/packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts#L50